log4j-core/src/main/java/org/apache/logging/log4j/core/appender/DefaultErrorHandler.java

```diff
@@ -84,14 +75,24 @@ public void error(final String msg, final Throwable t) {
      */
     @Override
     public void error(final String msg, final LogEvent event, final Throwable t) {
+        handleErrorMessage(msg, t);
+        if (!appender.ignoreExceptions() && t != null && !(t instanceof AppenderLoggingException)) {
+            throw new AppenderLoggingException(msg, t);
+        }
+    }
+
+    private void handleErrorMessage(
```

Comment:
All the parameters really should be on the method declaration since it isn't going to exceed the max line length.

Reply:
Fixed style and squashed with a reasonable commit name